index.html

```diff
@@ -1828,11 +1828,62 @@
       </section>
 
       <section>
-        <h3>Bearer Claims</h3>
+        <h3>Bearer Credentials</h3>
 
-        <p class="issue">
-Bearer claims containing PII or unique identifiers can be correlated.
-Bearer claims can be tracked based on usage patterns.
+        <p>
+A <em>bearer credential</em> is a privacy enhancing piece of information,
+such as a concert ticket, that entitles the <a>holder</a> of the claim to a
+particular resource without divulging sensitive information about the holder.
+        </p>
+
+        <p>
+Verifiable Credentials that are bearer credentials are possible by not
+specifying the <a>subject</a> identifier in the <code>claim</code> property.
+For example, the following Verifiable Credential is a bearer credential:
+        </p>
+
+        <pre class="example nohighlight" title="Usage of issuer properties">
+{
+  "id": "http://dmv.example.gov/credentials/temporary/28934792387492384",
+  "type": ["Credential", "ProofOfAgeCredential"],
+  "issuer": "https://dmv.example.gov/issuers/14",
+  "issued": "2017-10-22T12:23:48Z",
+  "claim": {
+    <span class="comment">// note that the 'id' property is not specified for bearer credentials</span>
```

Comment:
id property or bearer property as described above?  Needs a small bit of clarification.

Reply:
I added text above the example to clarify what the id property does.